http_server/routes/socketio.go

```diff
@@ -130,6 +130,15 @@ func getStationOverviewData(stationName string, h *server.Handlers) (models.Stat
 		return models.StationOverviewData{}, err
 	}
 
+	schema, err := h.Schemas.GetSchemaByStationName(station.Schema.SchemaName)
+
+	if err != nil {
+		return models.StationOverviewData{}, err
+	}
+
+	schemaVersion, err := h.Schemas.GetUsedSchemaVersion(station.Schema.UsingVersionNumber, schema)
+	schemaDetails := models.SchemaDetails{SchemaName: schema.Name, UsingVersionNumber: station.Schema.UsingVersionNumber, IsActiveVersionNumber: schemaVersion.Active}
```

Comment:
IsActiveVersionNumber => updatesAvailable - will be true if Active field is false